appyters/ChEA3_Appyter/appyter.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
   "name": "ChEA3_Appyter",
   "title": "ChEA3 Appyter",
-  "version": "0.0.1",
+  "version": "0.0.3",
```

Comment:
Versions should only be incremented per PR. Since there was no release ever, so far, it should be 0.0.1